client/core/core.go

```diff
@@ -2831,8 +2835,8 @@ func (c *Core) Register(form *RegisterForm) (*RegisterResult, error) {
 	c.log.Infof("Attempting registration fee payment to %s, account ID %v, of %d units of %s. "+
 		"Do NOT manually send funds to this address even if this fails.",
 		regRes.Address, dc.acct.id, regRes.Fee, regFeeAssetSymbol)
-	feeRateSuggestion := dc.fetchFeeRate(feeAsset.ID)
-	coin, err := wallet.PayFee(regRes.Address, regRes.Fee, feeRateSuggestion)
```

Comment:
For Register, I think we need to do a `dc.fetchFeeRate` like before because this `dexConnection` is not yet a tracked/persistent dex connection and thus is not used by `c.feeSuggestionAny`

```
2022-02-02 13:28:57.158 [INF] CORE: Connected to DEX server at 127.0.0.1:17273 and listening for messages.
2022-02-02 13:28:57.182 [INF] CORE: Attempting registration fee payment to bcrt1qp5wfcq48h6d63wyy9qz0awtpfqwwv4sm4gc9mc, account ID 0af1b5e58f6a4dfc2149c5a5a7beeba7b514d3cfb7d14d2ab0927223f3f4f108, of 20000000 units of btc. Do NOT manually send funds to this address even if this fails.
2022-02-02 13:28:57.182 [WRN] CORE[btc]: Unable to get optimal fee rate, using fallback of 100
```